build.sh

```diff
@@ -6,7 +6,7 @@
 # Dependencies include: build-essential libncurses5 libncurses-dev cmake zlib1g-dev
 set -e # exit on first error
 
-jobs=4
+jobs=$(nproc)
```

Comment:
Would be good to keep it as 4 since some machines might be preemptive and struggling with more cores as per our experience.

Reply:
limit to 4 at least

Reply:
Limited to 4 at most, or let us just leave it as 4 or make it configurable. Sometimes the compilation seems to be quite intense and will not be able to finish when competing with each other